webapp-django/crashstats/supersearch/views.py

```diff
@@ -240,7 +240,14 @@ def get_report_list_parameters(source):
     params = {}
 
     for key, value in source.items():
-        if not value:
+        if (
+            not value or (
+                # if value == [None]
+                isinstance(value, list) and
+                len(value) == 1 and
+                value[0] is None
```

Comment:
Is it OK to do ``if value == [None]``? Iirc doing ``smth == None`` is a bad thing to do, so I thought it would also be bad to do ``== [None]``.